_episodes/03-index-slice-subset.md

```diff
@@ -16,7 +16,10 @@ objectives:
     - "Locate subsets of data using masks."
     - "Describe BOOLEAN objects in Python and manipulate data using BOOLEANs."
 keypoints:
-    - "FIXME"
+    - "Data can be subset using column headings, numeric ranges , operators, or specific index locations."
+    - "Python uses 0-based indexing."
+    - "Data indexing, slicing and subsetting are common steps in data wrangling and exploration."
```

Comment:
what message are you trying to convey with this keypoint? It seems too general and not Python-specific

Reply:
I like that - this is the Python lesson, but tapping into a larger set of tasks and concerns.

Reply:
Thanks April. Yes, I was specifically trying to make a broader picture of why this is important. Maybe is could say  something like "The `pandas` library useful for data indexing, slicing and subsetting, which are common steps in data wrangling and exploration."

Reply:
1. I think keypoints should be specific to episodes
2. Indexing and slicing are not Pandas-specific.

Perhaps, you meant something like this
```
For data exploration and wrangling in Python you can use such powerful subsetting methods as indexing (accessing individual elements of complex data structures _via_ their index) and slicing (accessing several individual elements...)
```

Reply:
I don't disagree, but the keypoint is specific to what we're doing in the lesson. If we want it to be more pythonic, I don't not like what you've written, but its a bit wordy. Perhaps

```suggestion
    - "Pandas enables common data exploration steps such as data indexing, slicing and subsetting."
```

Reply:
Python native lists and tuples can be indexed and sliced too.
Pandas and numpy can do condition-based subsetting.

Reply:
Sure, but I don't think it's on us to emphasize the uniqueness of the technology we teach.

Reply:
`Pandas supports...` would make this keypoint more accurate.

On a different note: aren't indexing and slicing examples of subsetting? Or did we miss `conditional` before `subsetting`?

Reply:
I think we ought to change line 37 to `condition-based` or `conditional` subsetting

Reply:
yep! i love this new beta feature

Reply:
It really is great. It's exactly what I need for working with people who are new to LaTeX, too. Leaving line edits is always the biggest stumbling block for the new folks.